Client/Frontend/ContentBlocker/ContentBlockerSettingViewController.swift

```diff
@@ -71,6 +71,7 @@ class ContentBlockerSettingViewController: ContentBlockerSettingsTableView {
                 ContentBlockerHelper.prefsChanged()
 
                 LeanPlumClient.shared.track(event: .trackingProtectionSettings, withParameters: ["Enabled option": option.rawValue as AnyObject])
+                UnifiedTelemetry.recordEvent(category: .action, method: .change, object: .setting, value: ContentBlockerHelper.PrefKeyEnabledState, extras: ["to": option.rawValue])
```

Comment:
I think it might have to be `method: .settingChange, object: .trackingProtectionSetting`, as the value itself won't be clear what it applies to.